### PR TITLE
D-Link.D-View version 6.00.03B19

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108292)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
> Automatic Validation ended with: image

From the error, this looks like an upgrade package for the main D-View application. The InstallerUrl, appears to be from a Russian branch of the company. The version `8.0` package from D-Link's main site is available for free trial, but requires registration: 
image

Is there a main package URL that doesn't require registration?

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with:
> image

This is the official URL, from the official FTP site.

File: manifests/d/D-Link/D-View/6.00.03B19/D-Link.D-View.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: D-Link.D-View
+PackageVersion: 6.00.03B19
+PackageLocale: en-US
+Publisher: D-Link
+PackageName: D-View
```
Comment (stephengillie) [inline manifests/d/D-Link/D-View/6.00.03B19/D-Link.D-View.locale.en-US.yaml:8]:
```suggestion
PackageName: D-View
PackageUrl: https://ftp.dlink.ru/pub/Software
```

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Pull request closed without merging**